src/feditest/testplan.py

```diff
@@ -35,10 +35,6 @@ def load(filename: str) -> 'TestPlanConstellationNode':
         return msgspec.convert(testplanconstellationnode_json, type=TestPlanConstellationNode)
 
 
-    def __str__(self):
-        return self.name
-
-
```

Comment:
Is there some kind of Python behavior that I don't know about? Or you think we simply don't need this?

Reply:
I accidentally printed one of these objects and there was an exception. There's no `name` field defined in the type AFAICT.

Reply:
Oops.